source/partials/release_notes/_release-18-3-0.md.erb

```diff
@@ -13,25 +12,22 @@ This improves performance for large GoCD instances with thousands of pipelines.
 a javascript based frontend that is several thousand times faster on large instances, reducing the CPU usage of your browser. Your laptop battery will thank you. :)
 * reduces the polling rate when the dashboard is shown in a browser tab that is hidden.
 
-> Please report any issues that you observe on [GitHub issues](https://github.com/gocd/gocd/issues).
 
-
-<h4>Plugin Infrastructure Enhancements</h4>
-We have been working on a few improvements to our plugin infrastructure.
+<h4>Plugin Enhancements</h4>
+We have been working on a few improvements to our plugin architecture.
 
 **Errors and Warnings support:**
 We have a [new API](https://plugin-api.gocd.org/18.3.0/notifications/#add-server-health-messages) that allows plugins to add Errors and Warnings to the Go Server. These will show up along with the server's own Errors and Warnings. It will allow plugin developers to notify users if any errors or exceptions occur.
 
 <figure class='shrink-figure-75'>
   <%= image_tag 'releases/18.3.0/plugin-error-warning-messages.png', alt: 'Errors and Warnings from Plugins' %>
-  <caption>Errors and Warnings from Plugins</caption>
 </figure>
 
 
 **Multiple Extension Support for Plugins:**
 Plugins can now have [multiple extensions](https://github.com/gocd/gocd/pull/4123) be part of a single plugin JAR. This will allow plugin developers to create plugins with better functionality - for instance, the Notification and Task extension can be used in the same plugin.
```

Comment:
Change this to,

``` Plugins can now implement [multiple extensions](https://github.com/gocd/gocd/pull/4123). This will allow plugin developers to create a single plugin to support multiple extensions which complement each other. E.g This would allow a writing a GitHub PR workflow plugin which requires both SCM and Notification extensions. ```

Reply:
Small tweak: This would allow one to write a new GitHub PR workflow plugin with both SCM and Notification extensions.